OSSM-8511 Document how to delete OSSM 2.6 after upgrading to 3.0 (#87466)
openshift/openshift-docs: modules/ossm-migrating-complete-remove-2-6-operator-crds.adoc

// Module included in the following assemblies:
//
// * service-mesh-docs-main/migrating/done/ossm-migrating-complete-assembly

:_mod-docs-content-type: PROCEDURE
[id="ossm-migrating-complete-remove-2-6-operator-crds_{context}"]
= Remove the Service Mesh 2.6 operator and custom resource definitions

> Once you have migrated your deployment and workloads, you can remove the {SMproduct} 2.6 operator and custom resource definitions (CRDs).

Reviewer: ```suggestion
Once you have migrated your workloads and removed all of the resources from the procedure above, you can remove the {SMproduct} 2.6 operator and custom resource definitions (CRDs).
```

.Prerequisites

* You have completed your deployment and workloads migrations.

> * You are logged in to the {ocp-product-title} web console as a user with the cluster-admin role.

Reviewer: One more prereq is to finish procedure above "Remove the Service Mesh 2.6 control plane"

.Procedure

. Check that there are no {SMPRoductName} 2.6 resources left by running the following command:
+
[source,terminal]
----
$ oc get smcp,smm,smmr -A
----
+
.Example output
----
No resources found
----

. Remove the operator by running the following commands:

.. Find the operator subscription:
+
[source,terminal]
----
csv=$(oc get subscription servicemeshoperator -n openshift-operators -o yaml | grep currentCSV | cut -f 2 -d ':')
----

.. Delete the subscription:
+
[source,terminal]
----
$ oc delete subscription servicemeshoperator -n openshift-operators
----

> .. Delete the version:

Reviewer: ```suggestion
.. Delete the clusterserviceversion:
```

// Module included in the following assemblies:
//
// * service-mesh-docs-main/migrating/done/ossm-migrating-complete-assembly

:_mod-docs-content-type: PROCEDURE
[id="ossm-migrating-complete-remove-2-6-operator-crds_{context}"]
= Remove the Service Mesh 2.6 operator and custom resource definitions

> After you remove the {SMProductName} 2 `ServiceMeshControlplane` resource, and all other {SMProduct} 2 resources, you can remove the {SMproduct} 2.6 operator and custom resource definitions (CRDs).

Reviewer 2: ```suggestion
After you remove the {SMProductName} 2 `ServiceMeshControlplane` resource, and all other {SMProduct} 2 resources, you can remove the {SMproduct} 2.6 Operator and custom resource definitions (CRDs).
```

.Prerequisites

* You have completed migrating your workloads.
* You have completed migrating your gateways.
* You have removed the {SMProduct} 2 `ServiceMeshControlPlane` resource.
* You have removed all other {SMProduct} 2 resources.
* You are logged in to the {ocp-product-title} web console as a user with the cluster-admin role.

.Procedure

. Check that there are no {SMPRoductName} 2.6 resources left by running the following command:
+
[source,terminal]
----
$ oc get smcp,smm,smmr -A
----
+
.Example output
----
No resources found
----

> . Remove the operator by running the following commands:

Reviewer 2: ```suggestion
. Remove the Operator by running the following commands:
```

> .. Find the operator subscription:

Reviewer 2: ```suggestion
.. Find the Operator subscription:
```
